common/persistence/dataInterfaces.go

```diff
@@ -775,24 +775,24 @@ type (
 
 	// DomainInfo describes the domain entity
 	DomainInfo struct {
-		ID          string
-		Name        string
-		Status      int
-		Description string
-		OwnerEmail  string
+		ID          string `db:"domain_info_id"`
+		Name        string `db:"domain_info_name"`
+		Status      int    `db:"domain_info_status"`
+		Description string `db:"domain_info_description"`
+		OwnerEmail  string `db:"domain_info_owner_email"`
 	}
 
 	// DomainConfig describes the domain configuration
 	DomainConfig struct {
 		// NOTE: this retention is in days, not in seconds
-		Retention  int32
-		EmitMetric bool
+		Retention  int32 `db:"domain_config_retention"`
```

Comment:
You don't need to duplicate the qualifier in the tag name. i.e.  domain_config_retention -> retention. Same below.